packages/@aws-cdk/toolkit-lib/lib/actions/bootstrap/index.ts

```diff
@@ -0,0 +1,204 @@
+import { Tag } from '../../api/aws-cdk';
+
+/**
+ * Options for Bootstrap
+ */
+export interface BootstrapOptions {
+  readonly parameters?: BootstrappingParameters;
```

Comment:
Nice! I'm still thinking about the name here. WDYT? cc @kaizencc 

- `ToolkitStackParameters`
- `BootstrapParameters`
- `BootstrapStackParameters`
- `BootstrapDeploymentParameters`